src/components/editor/EditorPane.tsx

```diff
@@ -92,7 +95,7 @@ export class EditorPane extends React.Component<EditorPaneProps, {
     }
     let viewer;
     if (file) {
-      viewer = <Editor view={view} options={{ readOnly: file.isBufferReadOnly }} />;
+      viewer = <Editor ref={(ref) => this.setEditor(ref)} view={view} options={{ readOnly: file.isBufferReadOnly }} />;
```

Comment:
You don't need to grab a reference to the editor in order to trigger the save command. The `file` property in `EditorPane` should give you access to the active tab's file.